internal/local/deactivate_test.go

```diff
@@ -24,7 +27,7 @@ func TestDeactivation(t *testing.T) {
 	})
 
 	t.Run("returns UnableToDeactivate when ControlMode is pre-provisioning (0)", func(t *testing.T) {
-		lps := setupService(f)
+		lps := setupService(f2)
```

Comment:
if this is the only place f2 is used, i woudl suggest moving the creation/allocation of f2 into this test alone as its the only test it is used in. Alternatively, re-use f2 across all tests that need to fail, i.e. TestDeactivateACM. Info_test.go could be evaluated for that as well.

Reply:
done